webapp/portlet/src/main/webapp/WEB-INF/jsp/portlet/bannerLogo.jsp

```diff
@@ -51,7 +56,21 @@
 
     portalPath = portalConfigService.getUserHomePage(request.getRemoteUser());
     if (portalPath == null) {
-      portalPath = defaultHomePath;
+      SiteFilter siteFilter = new SiteFilter();
+      siteFilter.setExcludedSiteName(portalConfigService.getGlobalPortal());
+      siteFilter.setSiteType(SiteType.PORTAL);
+      siteFilter.setSortByDisplayOrder(true);
+      siteFilter.setFilterByDisplayed(true);
+      siteFilter.setDisplayed(true);
+      List<PortalConfig> portalConfigList = portalConfigService.getSites(siteFilter);
```

Comment:
If this is a filter that is used here and in `DefaultRequestHandler`, shouldn't we add a dedicated method for it with the adequate business logic ?

```suggestion
      List<PortalConfig> portalConfigList = portalConfigService.getDisplayedPotalConfigs();
```

Reply:
fixed